styles.css

```diff
@@ -156,8 +156,20 @@ hr {
     margin: 1rem 2rem;
 }
 
+.card-horizontal {
+    margin-left: 2rem;
+    margin-bottom: 1.25rem;
+}
+
+.card-vertical {
+    margin-left: 2rem;
+    margin-bottom: 1.25rem;
+}
```

Comment:
Since both classes have the same properties, you can simplify it to a single class called `card`